vet.sh

```diff
@@ -41,6 +41,11 @@ if [[ "$1" = "-install" ]]; then
   popd
   if [[ -z "${VET_SKIP_PROTO}" ]]; then
     if [[ "${GITHUB_ACTIONS}" = "true" ]]; then
+      if [[ -n "${VET_BUILD_GO_PROTOC}" ]]; then
```

Comment:
Let's add a comment here to mention why we need to build `protoc-gen-go-grpc`

Reply:
@arvindbr8 sure, make sense